install_config/imagestreams_templates.adoc

```diff
@@ -0,0 +1,326 @@
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+ifdef::openshift-enterprise[]
+Your OpenShift installation includes useful sets of Red Hat-provided
+link:../architecture/core_concepts/builds_and_image_streams.html#image-streams[image
+streams] and link:../architecture/core_concepts/templates.html[templates] to
+make it easy for developers to create new applications. By default, the
+link:../install_config/install/quick_install.html[quick installation] and
+link:../install_config/install/advanced_install.html[advanced installation]
+methods automatically create these sets in the *openshift* project, which is a
+default global project to which all users have view access. 
+endif::[]
+
+ifdef::openshift-origin[]
+You can populate your OpenShift installation with useful sets of
+Red Hat-provided
+link:../architecture/core_concepts/builds_and_image_streams.html#image-streams[image streams] and
+link:../architecture/core_concepts/templates.html[templates] to
+make it easy for developers to create new applications. By default, the
+link:../install_config/install/advanced_install.html[advanced installation] method automatically creates
+these sets in the *openshift* project, which is a default project to which all
+users have view access.
+endif::[]
+
+ifdef::openshift-enterprise[]
+The core set of image streams and templates are provided and supported by
+Red Hat with an active OpenShift Enterprise subscription for the following
+technologies:
+endif::[]
+ifdef::openshift-origin[]
+Image streams and templates are provided for the following technologies:
+endif::[]
+
+[horizontal]
+Languages::
+- link:../using_images/s2i_images/nodejs.html[Node.js]
+- link:../using_images/s2i_images/perl.html[Perl]
+- link:../using_images/s2i_images/php.html[PHP]
+- link:../using_images/s2i_images/python.html[Python]
+- link:../using_images/s2i_images/ruby.html[Ruby]
+Database::
+- link:../using_images/db_images/mongodb.html[MongoDB]
+- link:../using_images/db_images/mysql.html[MySQL]
+- link:../using_images/db_images/postgresql.html[PostgreSQL]
+Other Services::
+- link:../using_images/other_images/jenkins.html[Jenkins]
+
+ifdef::openshift-enterprise[]
+If you also have the relevant xPaaS Middleware subscription active on your
+account, image streams and templates are also provided and supported by Red Hat
+for each of following middleware services:
+
+[horizontal]
+Middleware Services::
+- link:../using_images/xpaas_images/eap.html[JBoss EAP]
+- link:../using_images/xpaas_images/a_mq.html[JBoss A-MQ]
+- link:../using_images/xpaas_images/jws.html[JBoss Web Server]
+- link:../using_images/xpaas_images/fuse.html[JBoss Fuse Integration Services]
+- link:../using_images/xpaas_images/decision_server.html[Decision Server]
+- link:../using_images/xpaas_images/data_grid.html[JBoss Data Grid]
+endif::[]
+
+[[before-you-begin-istemps]]
+== Before You Begin
```

Comment:
Changed "Before You Begin" into an admonition within the Overview. Was able to cut down on redundant explanation that way as well.